src/main/java/org/example/service/DeveloperService.java

```diff
@@ -33,30 +32,46 @@ public List<Developer> getDevelopers() {
                 ResultSet resultSet = statement.executeQuery(GET_DEVELOPERS)
         ) {
             while (resultSet.next()) {
-                int id = resultSet.getInt(COLUMN_ID);
-                String firstname = resultSet.getString(COLUMN_FIRST_NAME);
-                String lastname = resultSet.getString(COLUMN_LASTNAME);
-                int age = resultSet.getInt(COLUMN_AGE);
-                String proglang = resultSet.getString(COLUMN_PROG_LANG);
+                int id = resultSet.getInt(Developer.COLUMN_ID);
+                String firstname = resultSet.getString(Developer.COLUMN_FIRST_NAME);
+                String lastname = resultSet.getString(Developer.COLUMN_LAST_NAME);
+                int age = resultSet.getInt(Developer.COLUMN_AGE);
+                String programmingLanguage = resultSet.getString(Developer.COLUMN_PROGRAMMING_LANGUAGE);
 
                 Developer developer = new Developer();
                 developer.setId(id);
                 developer.setFirstName(firstname);
-                developer.setSecondName(lastname);
+                developer.setLastName(lastname);
                 developer.setAge(age);
-                developer.setProgLang(proglang);
+                developer.setProgrammingLanguage(programmingLanguage);
 
                 developers.add(developer);
             }
             LOGGER.info("{} developers have been retrieved from the database.", developers.size());
         } catch (SQLException ex) {
             LOGGER.error("Unable to retrieve developers from DB");
-            ex.printStackTrace();
-            throw new RuntimeException(ex);
         }
         return developers;
     }
 
+    public void addDeveloper(Developer developer) {
```

Comment:
What should we do, if there is an exception during the developer creation.
(Rename the method to `createDeveloper`)